modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/polling/ServiceOrderStatusChangePolling.java

```diff
@@ -55,16 +61,22 @@ public void fetchServiceOrderTaskStatusWithPolling(
         log.info("Start polling for service order status with order id: {}", orderId);
         AtomicReference<ServiceOrderStatusUpdate> ref =
                 new AtomicReference<>(new ServiceOrderStatusUpdate(previousKnownTaskStatus,
-                        FINAL_TASK_STATUS.contains(previousKnownTaskStatus), null));
+                        FINAL_TASK_STATUS.contains(previousKnownTaskStatus), null, null));
         try {
             Awaitility.await().atMost(pollingWaitPeriod, TimeUnit.SECONDS)
                     .pollDelay(0, TimeUnit.SECONDS) // first check runs without wait.
                     .pollInterval(pollingInterval, TimeUnit.SECONDS).until(() -> {
                         ServiceOrderEntity serviceOrderEntity = orderStorage.getEntityById(orderId);
                         TaskStatus taskStatus = serviceOrderEntity.getTaskStatus();
                         boolean isOrderCompleted = FINAL_TASK_STATUS.contains(taskStatus);
+                        Map<String, String> deployServiceProperties = new HashMap<>();
+                        if (taskStatus == TaskStatus.SUCCESSFUL) {
+                            DeployServiceEntity entity = deployServiceEntityHandler
+                                    .getDeployServiceEntity(serviceOrderEntity.getServiceId());
+                            deployServiceProperties = entity.getProperties();
+                        }
                         ref.set(new ServiceOrderStatusUpdate(taskStatus, isOrderCompleted,
-                                serviceOrderEntity.getErrorMsg()));
+                                serviceOrderEntity.getErrorMsg(), deployServiceProperties));
```

Comment:
this is not needed. The idea is, once the task is successful the UI must then use the "getSelfHostedServiceDetailsById/getSelfHostedServiceDetailsById" to fetch the details of the service properties.